src/coprocessor/dag/executor/aggregation.rs

```diff
@@ -69,26 +70,25 @@ impl AggrFunc {
     }
 }
 
-pub struct AggregationExecutor<'a> {
+pub struct AggregationExecutor {
     group_by: Vec<Expression>,
     aggr_func: Vec<AggrFuncExpr>,
-    group_keys: Vec<Rc<Vec<u8>>>,
-    group_key_aggrs: HashMap<Rc<Vec<u8>>, Vec<Box<AggrFunc>>>,
+    group_key_aggrs: OrderMap<Vec<u8>, Vec<Box<AggrFunc>>>,
```

Comment:
Could `group_key_aggrs` be defined as an `IntoIter`?

Reply:
It will be a big adjust, because `group_key_aggrs` is not constructed in `new`. If we define that as `IntoIter`, we can't use `entry API` when we build it.